scripts/screens/ChangeGenderScreen.py

```diff
@@ -0,0 +1,631 @@
+    pronouns_dict = ujson.load(f)
+
+
+class ChangeGenderScreen(Screens):
+
+    def __init__(self, name=None):
+        super().__init__(name)
+        self.next_cat_button = None
+        self.previous_cat_button = None
+        self.back_button = None
+        self.the_cat = None
+        self.selected_cat_elements = {}
+        self.buttons = {}
+        self.next_cat = None
+        self.previous_cat = None
+        self.elements = {}
+        self.windows = None
+        self.removalboxes_text = {}
+        self.removalbuttons = {}
+        self.pronoun_template = [
+            {
+                "ID": "Custom",
+                "subject": "",
+                "object": "",
+                "poss": "",
+                "inposs": "",
+                "self": "",
+                "conju": 1
+            }
+        ]
+        self.remove_button = {}
+        self.removalboxes_text = {}
+        self.boxes = {}
+        self.box_labels = {}
+        self.conju = 2
+
+    def handle_event(self, event):
+        if event.type == pygame_gui.UI_BUTTON_START_PRESS:
+            if event.ui_element == self.back_button:
+                self.change_screen("profile screen")
+            elif event.ui_element == self.next_cat_button:
+                if isinstance(Cat.fetch_cat(self.next_cat), Cat):
+                    game.switches["cat"] = self.next_cat
+                    self.update_selected_cat()
+                # else:
+                # print("invalid next cat", self.next_cat)
+            elif event.ui_element == self.previous_cat_button:
+                if isinstance(Cat.fetch_cat(self.previous_cat), Cat):
+                    game.switches["cat"] = self.previous_cat
+                    self.update_selected_cat()
+                # else:
+                # print("invalid previous cat", self.previous_cat)
+            elif event.ui_element == self.buttons["save"]:
+                if self.are_boxes_full():
+                    gender_identity = self.get_new_identity()
+                    self.the_cat.genderalign = gender_identity
+                    print("New Gender Identity Unlocked!")
+                    self.selected_cat_elements["identity_changed"].show()
+                    self.selected_cat_elements["cat_gender"].kill()
+                    self.selected_cat_elements["cat_gender"] = pygame_gui.elements.UITextBox(
+                        f"{self.the_cat.genderalign}",
+                        scale(pygame.Rect((252, 500), (500, 500))),
+                        object_id=get_text_box_theme("#text_box_30_horizcenter_spacing_95"),
+                        manager=MANAGER
+                    )
+
+            elif event.ui_element == self.buttons["add_pronouns"]:
+                PronounCreation(self.the_cat)
+                self.previous_cat_button.disable()
+                self.next_cat_button.disable()
+                self.back_button.disable()
+
+            else:
```

Comment:
Having a "else" statement for button event handling is confusing, and could possibly cause issues down the line.  It took me a while to figure out what this section of code was for.   If would be must clearer to use additional elif statements, rather than a blanket "else" statement for button clicks.

Reply:
I believe I fixed this, though I think when you fix the way the pronouns are identified this area of code might be changed as well.